### DISCUSSION TITLE
Redesigning the home page

### DISCUSSION THREAD

**Discussion by mindplay-dk** [Alex313031/thorium#850] [Ideas]
Hi @Alex313031 

I am learning design, and decided to use Thorium as a case study - I've been designing a new home page and logo, but I don't want to post everything in public yet.

If you'd like to see, could you reach out to me in private [on Twitter](https://x.com/mindplaydk) please? my DMs are open.

Not asking for anything in return, just figured maybe this could help promote Thorium 🙂

Now would be a good time to do it, I think - as Google are making bad decisions, and a lot of people are searching for a different browser right now, many of them wanting essentially Chrome minus the Evil. 🫠

**Comment by Alex313031** [comment]
@mindplay-dk Oooh, interesting, yes lets see! Ill try to message you on twitter
